README.md

```diff
@@ -39,45 +31,24 @@ In addition to the steps above there are a few addition steps required for trans
 
 #### New Language
 
-1. Create a folder using the 2 character code (e.g. ja, en, es, etc) with lesson subfolders:
+1. Create a folder using the ISO language code (e.g. ja, zh-hans, es, et al) with lesson subfolders:
 
   ```shell
   $ cd elixirschool
   $ mkdir -p ja/lessons/{basics,advanced,specifics,libraries}
-  $ touch ja/lessons/{basics,advanced,specifics,libraries}/.gitignore
+  $ touch ja/lessons/{basics,advanced,specifics,libraries}/.gitkeep
   ```
 
-1. Update `_config.yml` by including the 2 character code in `languages` and adding translations to `sections`, `description` and `toc`:
+1. Create a locale file for your new language using `_data/locales/en.yml` as a guide:
 
-  ```yaml
-  languages: ['en', 'ja']
-  default_lang: en
-  exclude_from_localization: []
-  sections:
-    - tag: basics
-      label:
-        en: Basics
-        ja: 基本
-
-  description:
-    en: Lessons about the Elixir programming language
-    ja: プログラミング言語Elixirのレッスン
-
-  toc:
-    en: Table of Contents
-    ja: 目次
+  ```shell
+  $ touch _data/locales/ja.yml
   ```
 
-1. If the new language is RTL (right-to-left) it should also be added to the `rtl_languages` list:
+1. If the new language is RTL (right-to-left) it should be added to the `rtl_languages` list in `config.yml`:
 
   ```yaml
-  rtl_languages: ['ar']
-  ```
-
-1. Add it to list in `index.md`:
-
-  ```markdown
-  Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl] [Português][pt], [Русском][ru] and [Bahasa Melayu][ms] and other.
+  script_direction: rtl
   ```
 
 #### Translated Lesson
```

Comment:
versioning instructions just wrong.

>     * `version` should consist of three digits: `major.minor.patch`, so:
>       * if this is a initial lesson translation, the version should be set to `1.0.0`;
>       * if you apply the original lesson updates to the translation, the version should be copied from the corresponding state of the original lesson;
>       * else bump one of the version numbers depending on how important is your change.

the version is given by the original English version. if they make changes or corrections, still they cannot bump the version number, and the version starts at whatever version the English version is at, not at 1.0.0
that applies for English lessons only